dali/operators/math/expressions/expression_impl_gpu.cuh

```diff
@@ -101,6 +100,31 @@ __device__ void ExecuteBinOp(Result *result, const Left *l, Right r, int64_t ext
   }
 }
 
+/**
+ * @brief Loop over tile of `extent` length and apply the ternary op
+ */
+template <ArithmeticOp op, typename Result, bool IsFirstTensor, bool IsSecondTensor,
+          bool IsThirdTensor>
+__device__ void ExecuteTernaryOp(Result *result,
+                                 expression_detail::param2_t<IsFirstTensor, Result> first,
```

Comment:
```suggestion
                                 expression_detail::const_param_t<IsFirstTensor, Result> first,
```
?

Reply:
I was thinking about something like `void_ptr_or_value_t` but I already had a quite narrow namespace over it, and don't want to make it any longer.